CMakeLists.txt

```diff
@@ -66,7 +66,11 @@ OPTION(DEPRECATE_HARD			"Do not include deprecated functions in the library"	OFF
    SET(REGEX_BACKEND			"" CACHE STRING "Regular expression implementation. One of regcomp_l, pcre2, pcre, regcomp, or builtin.")
 
 IF (UNIX)
-	OPTION(USE_NTLMCLIENT		"Enable NTLM support on Unix."				ON )
+	IF (NOT USE_HTTPS)
+	    OPTION(USE_NTLMCLIENT		"Enable NTLM support on Unix."				OFF )
+	ELSE()
+	    OPTION(USE_NTLMCLIENT		"Enable NTLM support on Unix."				ON )
+	ENDIF()
```

Comment:
It's quite verbose, but as far as I know there's no other way to do it due to `USE_HTTPS` accepting not only ON/OFF, but specific backends, too.